### DISCUSSION TITLE
Set Light and Dark themes based on time of day (or system)

### DISCUSSION THREAD

**Discussion by dessalines** [helix-editor/helix#6567] [Ideas] [answered]
It'd be really helpful to have some kind of a theme-mode switcher, based on the time of day. 

On vim, I was able to set startup themes using vimscript:

```vim
if strftime("%H") > 17 || strftime("%H") < 7  
  colo monokai
else
  set background=light
  colo PaperColor
endif
```

This wasn't ideal, because it only ran on startup.

**Comment by the-mikedavis** [comment, marked as answer]
https://github.com/helix-editor/helix/issues/2158
